REQUIRE

```diff
@@ -1 +1,2 @@
 julia 0.5
+Compat 0.18
```

Comment:
why 0.18? new-style overloading was implemented way back in 0.7.13 https://github.com/JuliaLang/Compat.jl/commit/fa91a2e41e7670b6ed1c8ff05e55eb323a107298